### PR TITLE
Change '2.0.0-alpha' to '2.0.0', the link from github/pamasse to github/zonemaster and noscript message

### PR DESCRIPTION
**Pull request by pamasse**
Minor changes:

- change '2.0.0-alpha' to '2.0.0'  
- change the link from github/pamasse to github/zonemaste
- update the noscript message  (displayed for no js client)

### CODE REVIEW

**Comment by pamasse** [comment]
I switch to Develop

**Comment by pamasse** [comment]
After reflection, i think that is a minor change and it must be incorporated in this release into the master branch. We don't want to propagate the new GUI with a wrong github url.

**Comment by matsduf** [comment]
I agree, but we should do that following the process. Minor things result in minor releases, i.e. 2.0.1. And instead of patching several times, we should see if there are more things. None of the issues will break, as far as we know.

**Comment by matsduf** [comment]
Please update the description so that it covers all changes.

**Comment by sandoche2k** [comment]
@pamasse pleae update

File: src/index.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 <noscript>
   <!-- If user's javascript is disabled -->
```
Comment (mattias-p) [inline src/index.html]: Have you considered removing this comment?

File: src/app/components/navigation/navigation.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-light bg-light fixed-top navbar-expand-lg" [ngClass]="{'shrink': isShrunk}" >
-  <a href="https://github.com/pamasse/zonemaster-gui" class="github-corner d-none d-lg-block d-xl-block" aria-label="View source on Github">
+  <a href="https://github.com/zonemaster/" class="github-corner d-none d-lg-block d-xl-block" aria-label="View source on Github">
```
Comment (mattias-p) [inline src/app/components/navigation/navigation.component.html]: Isn't it more correct to link to the zonemaster/zonemaster-gui repo?

Comment (pamasse) [reply src/app/components/navigation/navigation.component.html]: Go to zonemaster/zonemaster/README.md

File: src/index.html (hunk 1 above)
Comment (pamasse) [reply src/index.html]: Done

File: src/app/components/navigation/navigation.component.html (hunk 2 above)
Comment (pamasse) [reply src/app/components/navigation/navigation.component.html]: Done

**Comment by sandoche2k** [comment]
@matsduf please review

**Comment by sandoche2k** [comment]
@pamasse pls. merge

**Pull request merged**